### PR TITLE
DGC-78: Changed Lightning version in composer. Closes #78

### LINKED ISSUE #78: Upgrade to Lightning distro
https://www.drupal.org/project/lightning/releases/8.x-1.12

Does BLT or Composer do this for us automatically?

### PR DESCRIPTION
**Pull request by timwood**
Let me know if I missed anything.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -114,3 +114,6 @@ nbproject/*
 *.gz
 *.sql
 *.zip
+
+# Ignore cloudflare CMI export file because it includes API keys.
+config/default/cloudflare.settings.yml
```
Comment (aweingarten) [inline .gitignore]: No action required.  This is a very minor nitpick: I would break this out into a different commit since it represents a distinct concept from the rest of the commit.

Comment (timwood) [inline .gitignore]: Okay. Next time... ;-)
